### DISCUSSION TITLE
Starting now with Frigate, start with 0.8.0 beta?

### DISCUSSION THREAD

**Discussion by CasparTheBridge** [blakeblackshear/frigate#410] [Ask A Question] [answered]
I'm starting to install Frigate now. But I was checking the notes of the 0.8.0 beta and saw a lot of breaking changes, makes it more sense to start with the beta now so I can do an easy switch over to the next stable release?

Any plans when the new stable will be released?

**Comment by blakeblackshear** [comment, marked as answer]
I recommend starting with the beta. I am pushing a new beta 2 release today. New stable should be released within a few weeks.

**Reply by CasparTheBridge** [reply]
Great, thank you for your quick reply!
It looks good, interested to see how it preforms 👍
